guests/eth-block/src/main.rs

```diff
@@ -36,8 +76,50 @@ pub fn main() {
         .expect("Failed to execute transactions")
         .build::<BuildFromMemDbStrategy>()
         .expect("Failed to build the resulting block");
-    // Output the resulting block's hash to the journal
-    env::commit(&output.hash());
-    // Leak memory, save cycles
-    core::mem::forget(output);
+
+    // get hash of the block header and print it
+
+    let output_hash = output.hash();
+    let output_hash_str = output_hash.to_string();
+    println!("{}", output_hash_str);
+
+    // generate random Ethereum public & private keys
+
+    use secp256k1::rand::rngs::OsRng;
+    use secp256k1::{Secp256k1, Message};
+    use secp256k1::hashes::sha256;
+
+    let secp = Secp256k1::new();
+    let (secret_key, public_key) = secp.generate_keypair(&mut OsRng);
```

Comment:
What do you think is possible/best:
1. Restarting the program each time and storing/loading the private key from disk
2. Keeping the program running and just keeping the private key in memory (but if the program stops, well, have to recover anyway somehow)

I don't know if restarting the program is even possible? Does the attestation changes when you do that?